### ISSUE TITLE
[2.4 rc4 + fullaudio] video streams crash when disconnecting and reconnecting audio

### ISSUE THREAD

**Issue by schrd** [bigbluebutton/bigbluebutton#13687]
**Describe the bug**
On a server with fullaudio enabled, videos freeze and stream crashes if audio connection is disabled and enabled again.

**To Reproduce**
You need multiple participants.

1. each participant joins using microphone
2. each participant enables a camera
3. all participants should do activity visible in video
4. one participant disconnects audio
5. video from the participant will freeze for the other participants
6. reconnect audio
7. video from other participants will freeze for the user who disconnected audio
8. user who disconnected audio will get a exception:

~~~
Uncaught (in promise) TypeError: peer is undefined
    handlePubPeerCreation component.jsx:511
    bindEnvironment meteor.js:1236
~~~
I think the bug is triggered here:
https://github.com/bigbluebutton/bigbluebutton/blob/314f35c44eb12bc5f6827469d3ca61365a9da4ab/bigbluebutton-html5/imports/ui/components/video-provider/component.jsx#L511

**BBB version:**
BBB 2.4 commit 314f35c44eb12bc5f6827469d3ca61365a9da4ab

**Desktop (please complete the following information):**
Linux, Firefox ESR

**Comment by prlanzarin** [comment]
> I think the bug is triggered here:

Bug is server side; that error log is a collateral of a server side ejection and is harmless.

**Comment by mariogasparoni** [comment]
Thanks @schrd , we are going to check this in our end

**Comment by prlanzarin** [comment]
Fixed in [v2.4-rc.5](https://github.com/bigbluebutton/bigbluebutton/releases/tag/v2.4-rc-5)
